benchmark/test_runner/global-concurrent-tests.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+const common = require('../common');
+const { it } = require('node:test');
+
+const bench = common.createBenchmark(main, {
+  n: [100, 1000, 1e4],
+  type: ['sync', 'async'],
+}, {
+  // We don't want to test the reporter here
+  flags: ['--test-reporter=./benchmark/fixtures/empty-test-reporter.js'],
+});
+
+async function run(n, type) {
+  const promises = new Array(n);
+  switch (type) {
+    case 'sync':{
+      for (let i = 0; i < n; i++) {
+        promises[i] = it(`${i}`, () => {
+        });
+      }
+      break;
+    }
+
+    case 'async':
+      for (let i = 0; i < n; i++) {
+        promises[i] = it(`${i}`, async () => {
+        });
```

Comment:
I don't have any references, but I think this function can be optimized by v8 depending on how many times this will be called, and the optimization can affect the results to be equal to the empty function, I would add something like for all these empty functions:

```js
let avoidV8Optimization;

promises[i] = it(`${i}`, async () => {
  avoidV8Optimization = i;
});
```

Reply:
thanks, What kind of optimization we wanna avoid again? removing the `async`?